### PR TITLE
Improve wording in the README, Post-Install and Pre-Install READMEs

### PR DESCRIPTION
**Pull request by ghost**
I think it will be pretty good if we improve the wording by putting words in bold, italic, underline. Also, I've put some words in code blocks like `sudo` or `systemd-resolved` 😀

### CODE REVIEW

**Comment by RoyalOughtness** [comment]
some of these changes are fine, others are not. secureblue's branding is lowercase, for example.

File: README.md [hunk 1]
```diff
@@ -4,60 +4,59 @@
...
 [![Donate](https://img.shields.io/badge/Donate-blue.svg)](https://github.com/secureblue/secureblue/blob/live/DONATE.md)
 
-This repo uses [BlueBuild](https://blue-build.org/) to generate hardened operating system images, using [uBlue](https://universal-blue.org)'s [Fedora Atomic](https://fedoraproject.org/atomic-desktops/)-based [base images](https://github.com/orgs/ublue-os/packages?repo_name=main) as a starting point. This hardening is done with the following goals in mind:
+This repo uses [BlueBuild](https://blue-build.org/) to *generate* hardened operating system images, using [uBlue](https://universal-blue.org)'s [Fedora Atomic](https://fedoraproject.org/atomic-desktops/)-based [base images](https://github.com/orgs/ublue-os/packages?repo_name=main) as a starting point. This hardening is done with the following goals in mind:
 
-- Increase defenses against the exploitation of both known and unknown vulnerabilities.
-- Avoid sacrificing usability for most use cases where possible
+- **Increase** defenses against the exploitation of *both* **known** and **unknown** vulnerabilities.
```
Comment (RoyalOughtness) [inline README.md]: all this bolding and italicizing is excessive and makes it more confusing

File: README.md [hunk 2]
```diff
@@ -4,60 +4,59 @@
...
 
-The following are not in scope for this project:
-- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for security theater. This is especially true when at odds with improving security.
-- Anything related to "degoogling" chromium. For example, we will not be replacing chromium with Brave or ungoogled-chromium.
+The following are **not** in scope for this project:
+- Anything related to "**privacy**", since Fedora is *already* sufficiently private and "privacy" often serves as a euphemism for the security theater. This is *especially* true when at odds with improving security.
+- Anything related to "*degoogling*" chromium. For example, we will **not** be replacing Chromium with Brave or ungoogled-chromium.
 
-## What
+## What hardening is made under the hood?
```
Comment (RoyalOughtness) [inline README.md]: this was fine how it was

File: README.md [hunk 3]
```diff
@@ -102,113 +101,123 @@ While it's recommended to use a Fedora Atomic iso to install and then rebase tha
...
+- `aurora-main-hardened`
+- `aurora-nvidia-hardened`
+- `silverblue-main-hardened`
+- `silverblue-nvidia-hardened`
+- `sericea-main-hardened`
+- `sericea-nvidia-hardened`
+
+###### Experimental
+
+- `cinnamon-main-hardened`
```
Comment (RoyalOughtness) [inline README.md:118]: adding the backticks is nice, the rest I don't understand

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]
see above

File: POSTINSTALL-README.md [hunk 4]
```diff
@@ -55,21 +55,21 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
...
 > [!NOTE]
-> We log in as admin to do the final step of removing the user account's wheel privileges in order to make the operation of removing those privileges depend on having access to your admin account, and the admin account functioning correctly first.
+> We log in as admin to do the final step of removing the user account's `wheel` privileges in order to make the operation of removing those privileges depend on having access to your admin account, and the admin account functioning correctly first.
 
 5. Log in as `admin`
 6. ```gpasswd -d {your username here} wheel```
 7. ```reboot```
 
-When not in the wheel group, a user can be added to a dedicated group, otherwise certain actions are blocked:
+When not in the `wheel` group, a user can be added to a dedicated group, otherwise certain actions are blocked:
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:64]: these are good

File: POSTINSTALL-README.md [hunk 5]
```diff
@@ -94,7 +94,7 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
 
 1. Go to [uBlock Origin Lite](https://chromewebstore.google.com/detail/ublock-origin-lite/ddkjiahejlhfcafbddmgiahcphecmpfh?pli=1) ([Why Lite?](https://developer.chrome.com/docs/extensions/develop/migrate/improve-security))
 2. Install it
-3. In the extension's settings, make sure all of the lists under Default and Miscellaneous are checked (and at your preference, lists in the Annoyances section or country-specific lists)
+3. In the extension's settings, make sure all of the lists under "Default" and "Miscellaneous" are checked (and at your preference, lists in the "Annoyances" section or country-specific lists)
```
Comment (RoyalOughtness) [inline POSTINSTALL-README.md:97]: keep this

File: PREINSTALL-README.md [hunk 6]
```diff
@@ -1,16 +1,16 @@
-# secureblue
+# Secureblue
 
-The recommended method to install secureblue is to rebase from an upstream silverblue/kinoite installation. Before rebasing and during the installation, the following checks are recommended.
+The recommended method to install SecureBlue is to rebase from an upstream Silverblue/Kinoite installation. *Before* rebasing and during the installation, the following checks are __recommended.__
```
Comment (RoyalOughtness) [inline PREINSTALL-README.md]: revert these

File: PREINSTALL-README.md [hunk 7]
```diff
@@ -1,16 +1,16 @@
...
 
-## Preinstall guide
+## Pre-Install Guide
```
Comment (RoyalOughtness) [inline PREINSTALL-README.md:5]: this and below is fine

File: README.md [hunk 8]
```diff
@@ -4,60 +4,59 @@
   </a>
 </p>
 
-<h1 align="center">secureblue</h1>
+<h1 align="center">Secureblue</h1>
```
Comment (RoyalOughtness) [inline README.md]: lowercase secureblue is correct

File: README.md [hunk 9]
```diff
@@ -4,60 +4,59 @@
...
 [![Discord](https://img.shields.io/discord/1202086019298500629?style=flat&logo=discord&logoColor=white&label=Discord&labelColor=%235F6AE9&color=%2333CB56)](https://discord.com/invite/qMTv5cKfbF)
 [![Donate](https://img.shields.io/badge/Donate-blue.svg)](https://github.com/secureblue/secureblue/blob/live/DONATE.md)
 
-This repo uses [BlueBuild](https://blue-build.org/) to generate hardened operating system images, using [uBlue](https://universal-blue.org)'s [Fedora Atomic](https://fedoraproject.org/atomic-desktops/)-based [base images](https://github.com/orgs/ublue-os/packages?repo_name=main) as a starting point. This hardening is done with the following goals in mind:
+This repo uses [BlueBuild](https://blue-build.org/) to *generate* hardened operating system images, using [uBlue](https://universal-blue.org)'s [Fedora Atomic](https://fedoraproject.org/atomic-desktops/)-based [base images](https://github.com/orgs/ublue-os/packages?repo_name=main) as a starting point. This hardening is done with the following goals in mind:
```
Comment (RoyalOughtness) [inline README.md]: remove these italicizations and boldings

File: README.md [hunk 10]
```diff
@@ -4,60 +4,59 @@
...
-- Increase defenses against the exploitation of both known and unknown vulnerabilities.
-- Avoid sacrificing usability for most use cases where possible
+- **Increase** defenses against the exploitation of *both* **known** and **unknown** vulnerabilities.
+- **Avoid** sacrificing *usability* for most use cases *where* possible.
 
-The following are not in scope for this project:
-- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for security theater. This is especially true when at odds with improving security.
-- Anything related to "degoogling" chromium. For example, we will not be replacing chromium with Brave or ungoogled-chromium.
+The following are **not** in scope for this project:
+- Anything related to "**privacy**", since Fedora is *already* sufficiently private and "privacy" often serves as a euphemism for the security theater. This is *especially* true when at odds with improving security.
```
Comment (RoyalOughtness) [inline README.md]: and all these italicizations and boldings

File: README.md [hunk 11]
```diff
@@ -4,60 +4,59 @@
...
+Hardening __applied:__
 
-- Setting numerous hardened sysctl values (Inspired by but not the same as Kicksecure's) <sup>[details](https://github.com/secureblue/secureblue/blob/live/config/files/usr/etc/sysctl.d/hardening.conf)</sup>
-- Disabling coredumps in limits.conf
-- Disabling all ports and services for firewalld
-- Adds per-network MAC randomization
-- Blacklisting numerous unused kernel modules to reduce attack surface <sup>[details](https://github.com/secureblue/secureblue/blob/live/config/files/usr/etc/modprobe.d/blacklist.conf)</sup>
+- Setting numerous hardened sysctl values (Inspired by but not the same as Kicksecure's). <sup>[details](https://github.com/secureblue/secureblue/blob/live/config/files/usr/etc/sysctl.d/hardening.conf)</sup>
+- Disabling coredumps in `limits.conf`.
+- Disabling all ports and services for `firewalld`.
```
Comment (RoyalOughtness) [inline README.md:29]: these are fine

File: README.md [hunk 12]
```diff
@@ -4,60 +4,59 @@
...
+- Disable KDE GHNS by default. <sup>[why?](https://blog.davidedmundson.co.uk/blog/kde-store-content/)</sup>
+- Use HTTPS for all RPM mirrors.
+- Set all default container policies to `reject`, `signedBy`, or `sigstoreSigned`.
+- Remove SUID-root from [numerous binaries](https://github.com/secureblue/secureblue/blob/live/config/scripts/removesuid.sh) and replace functionality [using capabilities](https://github.com/secureblue/secureblue/blob/staging/config/files/usr/bin/setcapsforunsuidbinaries).
+- (Non-userns variants) Disabling unprivileged user namespaces.
+- (Non-userns variants) Replacing `bubblewrap` with `bubblewrap-suid` so Flatpak can be used without unprivileged user namespaces.
+
+## Why Fedora ?
+
+[Fedora](https://getfedora.org) is one of the *few* distributions that ships with [SELinux](https://selinuxproject.org/page/Main_Page) and associated tooling built-in and enabled **by default**. This makes it advantageous as a starting point for building a hardened system. However, out of the box it's **lacking** hardening in numerous other areas. This project's goal is to improve on that __significantly.__
```
Comment (RoyalOughtness) [inline README.md]: remove these italicizations

**Comment by ghost** [comment]
@qoijjj I pushed a new commit. If you see anything that should be changed, let me know!

File: README.md [hunk 13]
```diff
@@ -14,50 +14,49 @@
 This repo uses [BlueBuild](https://blue-build.org/) to generate hardened operating system images, using [uBlue](https://universal-blue.org)'s [Fedora Atomic](https://fedoraproject.org/atomic-desktops/)-based [base images](https://github.com/orgs/ublue-os/packages?repo_name=main) as a starting point. This hardening is done with the following goals in mind:
 
 - Increase defenses against the exploitation of both known and unknown vulnerabilities.
-- Avoid sacrificing usability for most use cases where possible
+- Avoid sacrificing usability for most use cases where possible.
 
 The following are not in scope for this project:
-- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for security theater. This is especially true when at odds with improving security.
-- Anything related to "degoogling" chromium. For example, we will not be replacing chromium with Brave or ungoogled-chromium.
+- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for the security theater. This is especially true when at odds with improving security.
```
Comment (RoyalOughtness) [inline README.md:20]: this was correct, your change makes it grammatically incorrect

File: README.md [hunk 14]
```diff
@@ -14,50 +14,49 @@
...
+- Anything related to "degoogling" Chromium. For example, we will not be replacing Chromium with Brave or ungoogled-chromium.
 
 ## What
 
-Hardening applied:
+Hardening __applied:__
```
Comment (RoyalOughtness) [inline README.md:25]: remove the emphasis here

File: README.md [hunk 15]
```diff
@@ -14,50 +14,49 @@
...
+- Disable passwordless `sudo` for `rpm-ostree install` <sup>[why?](https://github.com/rohanssrao/silverblue-privesc).
+- Brute force protection by locking user accounts for 24 hours after 50 failed login attempts, hardened password encryption and password quality suggestions.
+- Installing `usbguard` and `bubblejail`.
+- Set `opportunistic` DNSSEC and `DNSOverTLS` for `systemd-resolved`.
+- Configure `chronyd` to use Network Time Security (NTS). <sup>[using chrony config from GrapheneOS](https://github.com/GrapheneOS/infrastructure/blob/main/chrony.conf)</sup>
+- Disable KDE GHNS by default. <sup>[why?](https://blog.davidedmundson.co.uk/blog/kde-store-content/)</sup>
+- Use HTTPS for all RPM mirrors.
+- Set all default container policies to `reject`, `signedBy`, or `sigstoreSigned`.
+- Remove SUID-root from [numerous binaries](https://github.com/secureblue/secureblue/blob/live/config/scripts/removesuid.sh) and replace functionality [using capabilities](https://github.com/secureblue/secureblue/blob/staging/config/files/usr/bin/setcapsforunsuidbinaries).
+- (Non-userns variants) Disabling unprivileged user namespaces.
```
Comment (RoyalOughtness) [inline README.md:48]: you can remove the period from these, they're not sentences

File: README.md [hunk 16]
```diff
@@ -14,50 +14,49 @@
...
+- (Non-userns variants) Replacing `bubblewrap` with `bubblewrap-suid` so Flatpak can be used without unprivileged user namespaces.
+
+## Why Fedora ?
+
+[Fedora](https://getfedora.org) is one of the few distributions that ships with [SELinux](https://selinuxproject.org/page/Main_Page) and associated tooling built-in and enabled **by default**. This makes it advantageous as a starting point for building a hardened system. However, out of the box it's **lacking** hardening in numerous other areas. This project's goal is to improve on that __significantly.__
```
Comment (RoyalOughtness) [inline README.md:53]: undo the emphasis added

File: README.md [hunk 17]
```diff
@@ -14,50 +14,49 @@
...
 
 For more info on uBlue and BlueBuild, check out the [uBlue homepage](https://universal-blue.org/) and the [BlueBuild homepage](https://blue-build.org/).
 
 ## Customization
 
-If you want to add your own customizations on top of secureblue, you are advised strongly against forking. Instead, create a repo for your own image by using the [BlueBuild template](https://github.com/blue-build/template), then change your `base-image` to a secureblue image. This will allow you to apply your customizations to secureblue in a concise and maintainable way, without the need to constantly sync with upstream. 
+*If* you want to add your own customizations on top of Secureblue, you are advised **strongly** against forking. Instead, create a repo for your own image by using the [BlueBuild template](https://github.com/blue-build/template), then change your `base-image` to a Secureblue image. This will allow you to apply your customizations to Secureblue in a concise and maintainable way, without the need to constantly sync with upstream.
```
Comment (RoyalOughtness) [inline README.md:59]: remove these changes. secureblue is lowercase and the emphasis doesn't make sense

File: README.md [hunk 18]
```diff
@@ -92,7 +91,7 @@ To rebase a Fedora Atomic installation, choose an $IMAGE_NAME from the [list bel
 
 ### ISO 
 
-While it's recommended to use a Fedora Atomic iso to install and then rebase that installation to secureblue, you can also generate an iso and install that directly using [this script](generate_secureblue_iso.sh). Please note you should still follow the [post-install steps](README.md#post-install) when installing from a generated iso:
+While it's recommended to use a Fedora Atomic ISO file to install and then rebase that installation to Secureblue, you can also generate an ISO file and install that directly using [this script](generate_secureblue_iso.sh). Please note you should still follow the [post-install steps](README.md#post-install) when installing from a generated ISO:
```
Comment (RoyalOughtness) [inline README.md:94]: these changes can be reverted

File: README.md [hunk 19]
```diff
@@ -104,111 +103,111 @@ While it's recommended to use a Fedora Atomic iso to install and then rebase tha
...
+### Post-Install
 
 After installation, [yafti](https://github.com/ublue-os/yafti) will open. Make sure to follow the steps listed carefully and read the directions closely.
 
 Have a look at [POSTINSTALL-README](POSTINSTALL-README.md).
 
 #### Nvidia
-If you are using an nvidia image, run this after installation:
+
+If you are using an Nvidia image, run this after installation:
```
Comment (RoyalOughtness) [inline README.md:210]: `-nvidia` image

File: README.md [hunk 20]
```diff
@@ -217,8 +216,9 @@ rpm-ostree kargs \
     --append-if-missing=nvidia-drm.modeset=1
 ```
 
-#### Nvidia optimus laptop
-If you are using an nvidia image on an optimus laptop, run this after installation:
+#### Nvidia Optimus laptop
+
+If you are using an Nvidia image on an Optimus laptop, run this after installation:
```
Comment (RoyalOughtness) [inline README.md:221]: same here

File: README.md [hunk 21]
```diff
@@ -231,7 +231,10 @@ Follow the [contributing documentation](CONTRIBUTING.md#contributing), and make
...
 
-For local Development [building locally](CONTRIBUTING.md#building-locally) is the recommended approach.
+For local development, [building locally](CONTRIBUTING.md#building-locally) is the recommended approach.
+
+### Community
 
-### Community 
 Opening issues is preferred, but [Discord](https://discord.gg/qMTv5cKfbF) is available as well.
+
+[def]: https://getfedora.org
```
Comment (RoyalOughtness) [inline README.md:240]: what is this?

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

File: README.md [hunk 22]
```diff
@@ -14,50 +14,49 @@
...
 
 - Increase defenses against the exploitation of both known and unknown vulnerabilities.
-- Avoid sacrificing usability for most use cases where possible
+- Avoid sacrificing usability for most use cases where possible.
 
 The following are not in scope for this project:
-- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for security theater. This is especially true when at odds with improving security.
-- Anything related to "degoogling" chromium. For example, we will not be replacing chromium with Brave or ungoogled-chromium.
+- Anything related to "privacy", since Fedora is already sufficiently private and "privacy" often serves as a euphemism for the security theater. This is especially true when at odds with improving security.
+- Anything related to "degoogling" Chromium. For example, we will not be replacing Chromium with Brave or ungoogled-chromium.
```
Comment (RoyalOughtness) [inline README.md:21]: this is out of date, you need to rebase

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

**Comment by RoyalOughtness** [comment]
closing due to >1week with no response, incorporated some of the changes

**Pull request closed without merging**